source/pom.xml

```diff
@@ -69,7 +69,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <encoding>${source.encoding}</encoding>
```

Comment:
Hmm, but this says "Default value is: `${project.build.sourceEncoding}`.", which is the other way round (`encoding` takes the value of `project.build.sourceEncoding`).
Or am I misinterpreting?